### ISSUE TITLE
adjetivos vs substantivos

### ISSUE THREAD

**Issue by arademaker** [LFG-PTBR/MorphoBr#69]
Este issue é para uma discussão mais ampla que #57 (que trata de um caso específico). E continua discussão iniciada no issue #65 mas que também tem foco diferente.

Em resposta ao [comentário](https://github.com/LFG-PTBR/MorphoBr/issues/65#issuecomment-584908070) do @leoalenc:

Concordo que [elipses](https://en.wikipedia.org/wiki/Ellipsis_(linguistics)) (link para os mais novos) não devam sugerir entradas no léxico, mas este critério de corte não me parece óbvio. Na OWN-PT usamos alguns testes linguisticos. No inglês dois exemplos seriam:

> A good clue that you have an adjective is if you try to modify it with "very" or "rather" and it sounds ok: “very/rather favorable conditions” (ok) vs. “very/rather cotton shirts” (not ok). Another good clue is if you can make a comparative or superlative form out of it (“damper/dampest/more favorable/most favorable conditions” are all adjectives, but “cottoner/cottonest/more cotton/most cotton shirts” are not).

O primeiro parece funcionar para PT também.  Então só precisamos definir bem os testes para termos critérios para decisão.

**Comment by leoalenc** [comment]
@arademaker, concordo com você. Quanto mais testes tivermos, melhor. Os dois testes que você mencionou que funcionam para o inglês também servem para o português:

O carro barato foi feito na China, o caro na Alemanha.
O carro muito barato foi feito na China, o muito caro na Alemanha.
O carro mais barato foi feito na China, o mais caro na Alemanha.

Os exemplos mostram que temos na segunda oração um adjetivo com um substantivo subentendido, portanto, não precisamos incluir no léxico um substantivo “caro” nessa acepção. 
Seja agora o seguinte exemplo: 

*Você escutou a mais última?

Esse exemplo me parece agramatical, mostrando que temos a palavra última como substantivo, na acepção de última notícia, conforme dicionários citados em #57.
